walkthroughs/1A-integrate-event-and-api-driven-apps/walkthrough.adoc

```diff
@@ -1,6 +1,6 @@
-:walkthrough: 1A - Integrating message-oriented middleware with a RESTful API using AMQ Online
```

Comment:
@pwright is there a way to force this pattern to come before the SSO one? I named them 1A, 1B etc to force the ordering but would like to remove that naming style too.

Reply:
being discussed with Tiffany